classes/patterns/class-pageheader.php

```diff
@@ -58,8 +59,8 @@ public static function get_config( $params = [] ): array {
 					<div class="wp-block-media-text__content">
 						<!-- wp:group -->
 						<div class="wp-block-group">
-							<!-- wp:heading {"level":1,"placeholder":"' . __( 'Enter title', 'planet4-blocks-backend' ) . '","backgroundColor":"white"} -->
-							<h1 class="has-white-background-color has-background">' . $title_placeholder . '</h1>
+							<!-- wp:heading {"level":1,"placeholder":"' . __( 'Enter title', 'planet4-blocks-backend' ) . '","backgroundColor":"' . $background_color . '"} -->
+							<h1 class="has-' . $background_color . '-background-color has-background">' . $title_placeholder . '</h1>
```

Comment:
AH and I also see an issue because the block stays in the page container on its shorter side, so the bg color won't go on the complete width of the page, this needs more modifications :sob:

Reply:
Not necessarily, because it has a transparent background (instead of `white`) so it will take on whatever color it is put on top of (does that make sense?) But we could indeed add it to the Media & Text block too

Reply:
I've solved this issue [here](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/925/files#diff-7999c49c1872f6d5dc61d0c936d72f41dab22483b3a82b303ec6333c4d4756b8R43-R55) for example by using two Group blocks

Reply:
Oooh I see :bulb: would it make sense to include it to the Page header pattern instead ?

Reply:
Seems it's used with a background in all pattern layouts (I actually forgot it in the [High-Level Topic](https://jira.greenpeace.org/browse/PLANET-6737) one, oops 😬) so yeah, that would make sense!

Reply:
Do you think we should also make `grey-05` the default color then? 🤔

Reply:
yep, makes sense :+1:

Reply:
Updated!